### PR TITLE
Channel serialization options + e2e tests

### PR DESCRIPTION
**Pull request by sadym-chromium**

### CODE REVIEW

File: tests/test_channel.py [hunk 1]
```diff
@@ -149,3 +149,235 @@ async def test_channel_beforeAndAfterExecutionFinished(websocket, context_id):
...
+                        "localName": "body",
+                        "namespaceURI": "http://www.w3.org/1999/xhtml",
+                        "attributes": {}
+                    },
+                    "sharedId": ANY_SHARED_ID
+                }
+            ]],
+            "handle": ANY_STR
+        }),
+        # ({
```
Comment (thiagowfx) [inline tests/test_channel.py:262]: Could you remove these comments? Alternatively, if they are meant to exist, then keep them but skip the tests with `pytest.xfail(reason="TODO: fail")`

File: tests/test_channel.py [hunk 2]
```diff
@@ -149,3 +149,235 @@ async def test_channel_beforeAndAfterExecutionFinished(websocket, context_id):
...
+        #      "includeShadowTree": "open"
+        #  }, {}),
+        # ({
+        #      "includeShadowTree": "all"
+        #  }, {})
```
Comment (thiagowfx) [inline tests/test_channel.py:339]: same for these

**Review by thiagowfx — CHANGES_REQUESTED** [review]

**Comment by thiagowfx** [comment]
LGTM overall

File: tests/test_channel.py (hunk 1 above)
Comment (sadym-chromium) [reply tests/test_channel.py:262]: done

File: tests/test_channel.py (hunk 2 above)
Comment (sadym-chromium) [reply tests/test_channel.py:339]: done

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**